src/components/ApplicationGroup/Details/TriggerView/BulkCDTrigger.tsx

```diff
@@ -105,14 +112,12 @@ export default function BulkCDTrigger({
                         }
                     }
                 })
-                updateBulkInputMaterial(_materialListMap)
+                updateBulkInputMaterial(_cdMaterialResponse)
                 setUnauthorizedAppList(_unauthorizedAppList)
                 setLoading(false)
             })
             .catch((error) => {
-                if (!abortControllerRef.current.signal.aborted) {
```

Comment:
is removing this intentional?

Reply:
Yes. I've moved it under `_CDMaterialPromiseList.push()` operation